app/configurator/components/chart-options-selector.tsx

```diff
@@ -178,19 +178,12 @@ const EncodingOptionsPanel = ({
   const getFieldLabelHint = {
     x: t({ id: "controls.select.dimension", message: "Select a dimension" }),
     y: t({ id: "controls.select.measure", message: "Select a measure" }),
-    baseLayer: t({ id: "chart.map.layers.base", message: "Base Layer" }),
-    areaLayer: t({
-      id: "controls.select.measure",
-      message: "Select a measure",
-    }),
-    symbolLayer: t({
-      id: "controls.select.measure",
-      message: "Select a measure",
-    }),
-    segment: t({
-      id: "controls.select.dimension",
-      message: "Select a dimension",
-    }),
+    // FIXME: encoding types, so we don't need these there (chart options are
+    // handled in a separate file)
```

Comment:
Should the FIXME be dealt with in this PR ?

Reply:
I would prefer to handle this in another PR by the end of this week 🤞